lib/Sapi.php

```diff
@@ -113,6 +113,12 @@ public static function sendResponse(ResponseInterface $response): void
                     if ($copied <= 0) {
                         break;
                     }
+                    // Abort on client disconnect.
+                    // With ignore_user_abort(true), the script is not aborted on client disconnect.
+                    // To avoid reading the entire stream and dismissing the data afterward, check between the chunks if the client is still there.
+                    if (1 === ignore_user_abort() && 1 === connection_aborted()) {
```

Comment:
I wonder that `ignore_user_abort()` is defined as returning a bool, but the strict check against `1` seems to work?

https://www.php.net/manual/en/misc.configuration.php#ini.ignore-user-abort